syntax = "proto3";

package auth;

message TokenRequest {
    string token = 1;
}

message UserIdResponse {
    uint32 user_id = 1;
}

service AuthenticationService {
    rpc GetUserId(TokenRequest) returns (UserIdResponse);
}